src/ringmesh/mesh/CMakeLists.txt

```diff
@@ -37,12 +37,16 @@ target_sources(${target_name}
         "${lib_source_dir}/mesh_aabb.cpp"
         "${lib_source_dir}/mesh_builder.cpp"
         "${lib_source_dir}/mesh_index.cpp"
+        "${lib_source_dir}/mesh_base.cpp"        
+        "${lib_source_dir}/pointsetmesh.cpp"        
```

Comment:
Should we add underscore(s) in the file name?

Reply:
I would say yes to be consistent with other files.

Reply:
Ok